Source/DafnyCore/AST/Expressions/NestedMatchCaseStmt.cs

```diff
@@ -21,6 +21,14 @@ public class NestedMatchCaseStmt : NestedMatchCase, IAttributeBearingDeclaration
     this.Attributes = attrs;
   }
 
+  private NestedMatchCaseStmt(Cloner cloner, NestedMatchCaseStmt original) : base(original.tok, original.Pat) {
```

Comment:
Would be better if this calls a cloning constructor in the base class.

Reply:
I had that thought too, but can be improved later at least.